### ISSUE TITLE
Keyword separator behaviour (again, from a groups perspective)

### ISSUE THREAD

**Issue by oscargus** [JabRef/jabref#1877]
Let me rephrase this: the use of the keyword separator is ambiguous in JabRef. 

It is by default set to `, `. This leads to:
1. StringTokenizer splits on both `,` and ` `
2. when adding keywords `, ` is used as a separator

Clearly these two behaviors are just by luck sometimes compatible. Based on 1. it would be better to have `,` as the default.

In addition, the EntryUtil method for splitting on keywords used by BibEntry totally ignores this.

Relates to #705 

( @Siedlerchr 's comment was for the previous wording of this, although it is still relevant.)

I can in this context note that quite a bit of the refactoring I've made recently has been to pass this separator down to the correct methods. Except for the fact that I know that it is bad, I would almost suggest having the separator as a global static variable in model...

**Comment by oscargus** [comment]
I might not have been in that PR, but it is not working as expected anyway.

**Comment by oscargus** [comment]
OK, new info. It wasn't in that commit, but it seems like StringTokenizer is not handling spaces in the separator well...

**Comment by Siedlerchr** [comment]
String Tokenizer ist really really legacy code.  Should either be String.split or scanner
http://stackoverflow.com/questions/691184/scanner-vs-stringtokenizer-vs-string-split

**Comment by oscargus** [comment]
And in MedlineImporter the author decided to use `; ` as separator and ignore the preferences (the separator from the preferences is available in ImportFormatPreferences).

**Comment by koppor** [comment]
Refs #1950
